### DISCUSSION TITLE
Passphrase generator

### DISCUSSION THREAD

**Discussion by tobias-carlbom** [CorentinTh/it-tools#1228] [Ideas]
Hi,

Any plans on implementing a passphrase generator like Bitwarden password generator? [Link](https://bitwarden.com/password-generator/#password-generator)
<img width="1018" alt="Screenshot 2024-08-09 at 08 28 24" src="https://github.com/user-attachments/assets/1f817871-e4b7-4868-b64f-8e9bbb28e4c1">

Inspiration:
https://bitwarden.com/password-generator/#password-generator
https://github.com/HacKanCuBa/passphrase-py

Thanks!

**Comment by sharevb** [comment]
Hi @manprinsen and @CorentinTh , implemented in #1260

**Reply by tobias-carlbom** [reply]
awesome work, thanks :)

**Reply by sharevb** [reply]
Hi, if you arr interested in up to date version of it-tools, I made kind of a fork here : https://github.com/sharevb/it-tools (https://sharevb-it-tools.vercel.app/ and docker images https://github.com/sharevb/it-tools/pkgs/container/it-tools)

**Reply by viharm** [reply]
> Hi, if you arr interested in up to date version of it-tools, I made kind of a fork here : https://github.com/sharevb/it-tools (https://sharevb-it-tools.vercel.app/ and docker images https://github.com/sharevb/it-tools/pkgs/container/it-tools)

@sharevb , this is awesome, thanks. Is it possible to add numeric modifiers to your solution?

**Reply by sharevb** [reply]
Hi @viharm , yes, can add an option to add some numbers at end of words

**Reply by sharevb** [reply]
Hi @viharm and @carlbomsdata , implemented fully in my fork : https://sharevb-it-tools.vercel.app/passphrase-generator

**Comment by viharm** [comment]
There is already a javascript port of the excellent passphrase generator by Murray Grant. The port is created by Steven Zeck, and I'm really hoping that it is easy to integrate into IT Tools.

Original code: https://makemeapassword.ligos.net
Javascript port: https://saintly.zeck.net/readablepassphrase/

**Reply by sharevb** [reply]
Hi @viharm , yes it could be probably
